### ISSUE TITLE
Add site name to meta title tags 

### ISSUE THREAD

**Issue by mazil** [takeflight/wagtail-metadata#25]
Currently the meta title tags only use the page name by default. 

Apparently there are [some arguments against](https://yoast.com/page-titles-seo/#titles-social-media) including the site name in the social media meta tags, I think most of our sites could probably use the extra context. 

(Presumably these are just the default, and if a user cared enough they could still customise the Facebook and Twitter titles completely?)

TLDR: Suggestion: Make the meta title tags the same as `<title>` tag by default - i.e. include the site name - but still allow for overrides.

**Comment by timheap** [comment]
Using the new `get_meta_title()` method, the title can be overridden. As we usually include the `MetadataPageMixin` on our base `Page` model, we can customise this on a site-by-site basis (or even a page by page basis) with something like:

```python
class Page(MetadataPageMixin, wagtailmodels.Page):
    def get_meta_title(self):
        site = self.get_site()
        return '{} - {}'.format(self.title, site.site_name)
```

If you feel strongly that this should be the default behaviour, we can make that happen. People/sites who want the old behaviour of just the page title could implement `get_meta_title()` to restore the old behaviour.
